packages/create-block-tutorial-template/plugin-templates/$slug.php-dynamic.mustache

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
We probably don't need this file anymore.

Reply:
nice catch! I updated all of the files in the tutorial template to use the conditionals